shell.nix

```diff
@@ -2,6 +2,7 @@
 
 pkgs.mkShell {
 	buildInputs = with pkgs; [
+		git
```

Comment:
is this needed?

Reply:
sort of, it allows me to use graphite with the hub nix config. my machine's git version is not up to date enough to use it